src/reducers/storageobject/grid/index.js

```diff
@@ -0,0 +1,39 @@
+export const LOAD_SEVERAL = 'musit/storageobject-grid/LOAD_SEVERAL'
+export const LOAD_SEVERAL_SUCCESS = 'musit/storageobject-grid/LOAD_SEVERAL_SUCCESS'
+export const LOAD_SEVERAL_FAIL = 'musit/storageobject-grid/LOAD_SEVERAL_FAIL'
+
+const storageObjectGridReducer = (state = {}, action = {}) => {
+  switch (action.type) {
+    case LOAD_SEVERAL:
+      return {
+        ...state,
+        loading: true
```

Comment:
some places we have used loaded : false. We should have similar at all the places.